Software/Cpp/grove_dht_pro/grove_dht_example.cpp

```diff
@@ -0,0 +1,89 @@
+//
+// GrovePi Example for using the Grove Temperature & Humidity Sensor Pro
+// (http://www.seeedstudio.com/wiki/Grove_-_Temperature_and_Humidity_Sensor_Pro)
+//
+// The GrovePi connects the Raspberry Pi and Grove sensors.
+// You can learn more about GrovePi here:  http://www.dexterindustries.com/GrovePi
+//
+// Have a question about this example?  Ask on the forums here:  http://forum.dexterindustries.com/c/grovepi
+//
+/*
+## License
+
+   The MIT License (MIT)
+
+   GrovePi for the Raspberry Pi: an open source platform for connecting Grove Sensors to the Raspberry Pi.
+   Copyright (C) 2017  Dexter Industries
+
+   Permission is hereby granted, free of charge, to any person obtaining a copy
+   of this software and associated documentation files (the "Software"), to deal
+   in the Software without restriction, including without limitation the rights
+   to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+   copies of the Software, and to permit persons to whom the Software is
+   furnished to do so, subject to the following conditions:
+
+   The above copyright notice and this permission notice shall be included in
+   all copies or substantial portions of the Software.
+
+   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+   AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+   THE SOFTWARE.
+*/
+
+#include "grovepi.h"
+#include "grove_dht_pro.h"
+
+using GrovePi::DHT;
+using GrovePi::delay;
+using GrovePi::I2CError;
+
+//g++ -Wall grovepi.cpp grove_dht_pro.cpp grove_dht_example.cpp -o grove_dht_example.out -> without package installed
+//g++ -Wall -lgrovepicpp grove_dht_pro.cpp grove_dht_example.cpp -o grove_dht_example.out -> with package installed
```

Comment:
Will be better to mention the name of the package without version number, so that it will help users understand, which package we are talking about.

Also add sudo to g++ commands as they will throw a permission denied error and many will try to copy paste it directly